src/util/system.cpp

```diff
@@ -949,7 +949,7 @@ bool ArgsManager::ReadConfigFiles(std::string& error, bool ignore_invalid_keys)
             const size_t default_includes = add_includes({});
 
             for (const std::string& conf_file_name : conf_file_names) {
-                std::ifstream conf_file_stream{GetConfigFile(conf_file_name)};
+                std::ifstream conf_file_stream{GetConfigFile(fs::NormalizedPathFromString(conf_file_name))};
```

Comment:
In commit "util, refactor: Use GetPathArg to read "-conf" value" (88f48f39f84ae52f85819321dddf6e6bd9381be2)

I think it would be better to use `fs::PathFromString` not `fs::NormalizedPathFromString` here

- So this is a pure refactoring change that doesn't affect behavior
- To avoid changing user provided paths unnecessarily in a way that could make issues harder to debug
- To avoid interpreting paths unnecessarily, and let the OS handle it

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/24675#issuecomment-1105074421).